files/webdev-font/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "webdev-font",
+  "repo": "nickpolet/webdev-font",
+  "files": {
+    "include": ["css/webdev-font.css", "css/webdev-font.min.css", "fonts/webdev-font.eot", "fonts/webdev-font.ttf", "fonts/webdev-font.woff"]
```

Comment:
You need to add `./` before every path to retain the directory structure.
Also I dont think `css/webdev-font.css` is needed. The min should be enough